packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -138,19 +137,19 @@ it('visits a basic html page', () => {
       cy.get('.command-name-assert').should('have.length', 5)
 
       // (1) Assert Enabled
-      cy.get('.command-name-assert').should('contain.text', 'expect <button#increment> to be enabled')
+      cy.get('.command-name-assert').should('contain.text', 'expected <button#increment> to be enabled')
 
       // (2) Assert Visible
-      cy.get('.command-name-assert').should('contain.text', 'expect <button#increment> to be visible')
+      cy.get('.command-name-assert').should('contain.text', 'expected <button#increment> to be visible')
 
       // (3) Assert Text
-      cy.get('.command-name-assert').should('contain.text', 'expect <button#increment> to have text Increment')
+      cy.get('.command-name-assert').should('contain.text', 'expected <button#increment> to have text Increment')
 
       // (4) Assert Id
-      cy.get('.command-name-assert').should('contain.text', 'expect <button#increment> to have id increment')
+      cy.get('.command-name-assert').should('contain.text', 'expected <button#increment> to have id increment')
 
       // (5) Assert Attr
-      cy.get('.command-name-assert').should('contain.text', 'expect <button#increment> to have attr onclick with the value increment()')
```

Comment:
These were just wrong apparently? Assertions never show as 'expect', they show as 'expected'. I'm not sure how this was ever passing.